execution/executor-benchmark/src/transaction_generator.rs

```diff
@@ -290,12 +355,12 @@ impl TransactionGenerator {
                     sender.sequence_number,
                     &sender.private_key,
                     sender.public_key.clone(),
-                    encode_peer_to_peer_with_metadata_script(
-                        xus_tag(),
+                    experimental_stdlib::encode_transfer_token_between_galleries_script_function(
+                        tokens::bars_tag(),
                         receiver.address,
                         1, /* amount */
-                        vec![],
-                        vec![],
+                        sender.address,
+                        2,
```

Comment:
Quick question: could you clarify the workload setup?
The last two parameters of this function are defining the NFT ID to transfer `(creator, seq_num)`.
Are we guaranteed that the corresponding NFT ID is going to present at the sender's gallery? May be a quick comment in the code would be helpful.

Reply:
Yes, it is guaranteed to exist.